### PR TITLE
Feat: added list"

### PR DESCRIPTION
**Pull request by sadathshariff**
Added Simple List and Stacked List
List

### CODE REVIEW

**Review by ttahm3d — APPROVED** [review]
For now you can merge it. But if you ever want to refactor, use more semantic elements such as header, footer, section, main, and so on. Not every container has to be div. These semantic elements give a context to screenreaders (A11Y)

**Comment by sadathshariff** [comment]
> ese semantic elements give a context to scre

Thanks Tahir, will work on it later  while refactoring

File: CSS/Components/list.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.list-container {
+  width: 15rem;
+  height: max-content;
+  background-color: #fff;
```
Comment (shubhamkumar648) [inline CSS/Components/list.css]: you will also put variable colour from utility class

**Review by shubhamkumar648 — APPROVED** [review]
reaming looks good

**Pull request merged**